concepts/conditionals/about.md

```diff
@@ -0,0 +1,59 @@
+# About
+
+## Comparison operators
+
+R has a set of operators that is the same as many other languages:
+
+```R
+1 == 2 # FALSE
+1 != 2 # TRUE
+1 < 2 # TRUE
+1 > 2 # FALSE
+1 <= 2 # TRUE
+1 >= 2 # FALSE
+```
+
+There is no infix operator for exclusive or (i.e. A OR B, but not both). The `xor()` function can be used in this case.
+
+## Branching
+
+The basic form of an if-then-else is 
+
+```R
+if (x > 10) {
+    y <- "big"
+} else if (x > 2) {
+    y <- "medium"
+} else {
+    y <- "tiny"
+}
+```
+
+In this case, both parentheses `()` around the boolean and braces `{}` around the statement are required. 
+A simple `if` statement can be shortened:
+
+```R
+if (x == 0) return("success")
+```
+
+## Using vectors in conditionals
+
+Looking ahead to the `vector-filtering` concept: the conditional clause in parentheses `()` must evaluate to a single `TRUE`/`FALSE`. 
+Including vectors in the comparison usually gives a vector of booleans, so these will need to be wrapped in an aggregating function such as `all()` or `any()`.
```

Comment:
I think it would be helpful to follow this up with an example to illustrate.

e.g. 
```
numbers <- c(4,5,7,9,10,11)
if (any(numbers %% 3 == 0)) print("1 or more numbers are divisible by 3")
```

(maybe you can think of a better example)

Reply:
I've added a slightly extended version of this to `about.md`